src/auxiliary/math.jl

```diff
@@ -124,7 +124,7 @@ end
 end
 
 """
-    ln_mean(x, y)
+    Trixi.ln_mean(x::Real, y::Real)
```

Comment:
The functions in this file were not exported, so I clarified this in their docstrings.